travis/travis_run_tests

```diff
@@ -53,9 +53,15 @@ if __name__ == '__main__':
     tests_enabled = os.environ.get('TESTS') == '1'
     tests_unspecified = os.environ.get('TESTS') is None
     transifex_enabled = os.environ.get('TRANSIFEX') == '1'
+    is_oca_project = os.environ.get('TRAVIS_REPO_SLUG', '').startswith('OCA/')
+    is_oca_transifex_user = os.environ.get('TRANSIFEX_USER') == \
+        'transbot@odoo-community.org'
 
     # TRAVIS_PULL_REQUEST contains the pull request number or 'false'
     is_pull_request = os.environ.get('TRAVIS_PULL_REQUEST') != 'false'
+    # Avoid false red from forks using OCA transifex user
+    is_valid_transifex = not is_pull_request and \
+        is_oca_project == is_oca_transifex_user
```

Comment:
is_oca_project | is_oca_transifex_user | Description
---- | ---- | ----
True | True | Valid: It's ideal case where is a oca project and the tx user is oca too
True | False | **Valid: and will fail Transifex build**
False | True | Invalid: It's a fork or custom project using oca/mqt but using oca tx user  # Real reason of this feature
False | False | Valid: It's a fork or custom project using oca/mqt but using a custom oca tx user  # vauxoo projects (not oca forks) using mqt but with my own tx user for vauxoo